application/cypress/integration/card/card.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jest/expect-expect */
+/* eslint-disable jest/valid-expect */
+/// <reference types="cypress" />
+import { registrationHelper } from '../utils/registrationHelper';
+
+describe('Card should be checked', () => {
+  it('card reducer should be empty', () => {
+    registrationHelper();
+    cy.get('[data-cy=data-cy-app-cars]').click();
+    cy.get('[data-cy=data-cy-app-BMW]').click();
+    cy.get('[data-cy=data-cy-app-Volvo]').click();
+    cy.get('[data-cy=data-cy-app-cart]').click();
+    cy.window()
+      .its('store')
+      .invoke('getState')
+      .its('cartReducer')
+      .should('have.length', 2);
+    cy.window()
+      .its('store')
+      .invoke('getState')
+      .its('paymentCardReducer')
+      .should('deep.equal', {
+        surname: '',
+        name: '',
+        email: '',
+        mobile: '',
+        orderNum: '',
+        total: 0
+      });
```

Comment:
We should only check UI (what user see on the screen), because you can have correct data in the store, but rendering logic in incorrect. As a result UI will be broken, but your test will pass.
We have to check what user can see/click/etc

Reply:
fixed